Add/Remove/Delete Trigger Words

mdrammeh1 opened the pull request:
- ability to add/remove/edit triggers words
- custom list item widgets
- updated settings page to navigate to triggers page

mdrammeh1 committed:
Add/Remove/Delete Trigger Words

- ability to add/remove/edit triggers words
- custom list item widgets
- updated settings page to navigate to triggers page

chauntika reviewed (commented):
I reviewed the changes. I tried to run the program. It comes up where I can add a trigger word to the file. But when I type in the trigger word, it doesn't show up listed on the page. It may just be something I'm not doing on my computer. If it ran for you, then it may just be something I'm lacking on my computer. 
Let me know if it ran for you okay and all the functions worked. If it ran for you, then I will go ahead and approve the changes.

mdrammeh1 commented:
Yes it worked for me. It may be looking at the old assets folder for the trigger words

mdrammeh1 committed:
Corrected issue to initialize triggers file

- initialized triggers file
- removed references to model from triggers_view
- updated some print statements

chauntika reviewed (approved):
Good to go!
